omero/sysadmins/unix/install-web-trial.txt

```diff
@@ -0,0 +1,348 @@
+OMERO.web WSGI deployment (TRIAL)
+=================================
+
+OMERO.web is the web application component of the OMERO platform which
+allows for the management, visualization (in a fully multi-dimensional
+image viewer) and annotation of images from a web browser. It also
+includes the ability to manage users and groups.
+
+Deploying OMERO.web separately to OMERO.server is recommended as they
+perform best under different circumstances and require a different set of
+dependencies. This guide uses the example of deploying with nginx and
+gunicorn.
+
+Prerequisites
+-------------
+
+-  Python 2.7
+
+   -  :pypi:`virtualenv` (optional) tool to create isolated Python environments
+
+   -  :pypi:`PyPI <>` a package management system used to install and manage
+      software packages written in Python. PyPI is already installed if
+      you are using Python 2 >=2.7.9
+
+   -  `omego <https://github.com/ome/omego>`_ OME package management system
+
+   -  `ZeroC`_ IcePy 3.6 (IcePy 3.5 is also supported although requires global
+      installation from RPM)
```

Comment:
There's an interesting related issue that perhaps we might capture on a card or something: often for prerequisites (e.g. PostgreSQL) if some version (a) is the "obvious" natural one to install on my distro (e.g., what I get if I don't put a version number in the package name) and (b) does satisfy the prerequisites (at least minimum, or whatever) then I'll tend to install that rather than one the version *recommended* for software that needs it partly because (a) different software will tend to recommend different versions anyway and if I can satisfy all with one I'll do that and (b) when I have issues with it (e.g., in distro upgrade) I'd rather have the version that most other users of the distro have because then I'm more likely to find faster help. (Of course, I make exceptions depending on if the recommendation for above minimum is for reasons more concrete than, "newer so probably shinier too".) Again, I don't know if I'm unusual in this, but if not then it might have implications for our playbooks.